docs/_rtd/data_types.md

```diff
@@ -21,7 +21,29 @@ Used by: [calculate_channel_spatial_enrichment](https://ark-analysis.readthedocs
 [create_neighborhood_matrix](https://ark-analysis.readthedocs.io/en/latest/_markdown/ark.segmentation.html#ark.segmentation.marker_quantification.generate_cell_data)  
 Shape: 2D matrix of cells x cell_features  
 
-Description: This holds the extracted single cell data. Each row is a unique cell, and each column is a feature for that cell. This includes the counts for each marker, morphological information such as area and diameter, and information to link that cell back to the original image such as segmentation id and FOV.
+Description: This holds the extracted single cell data. Each row is a unique cell, and each column is a feature for that cell. This includes the counts for each marker, morphological information, and information to link that cell back to the original image such as segmentation id and FOV.  
+
+For each cell, the following morphology features calculated from `skimage.measure.regionprops` are included:
+
+* `cell_size`: the signal intensity. May not be the same as area depending on the signal extraction method used.
```

Comment:
I think cell size is just the size, can you confirm?

Reply:
It is the value of channel counts as returned by `signal_extraction.py`. We should probably rename it to something more descriptive, such as `channel_counts` or `signal_intensity`.

Reply:
Channel counts is the count for each channel. How could that be the value in a single column in the cell table?
This is where cell_size is defined: https://github.com/angelolab/ark-analysis/blob/db9065d18aab74aad0bfea6e275eede661b7a85a/ark/segmentation/marker_quantification.py#L119

Reply:
@ngreenwald OK, I got confused between cell size and cell (channel) counts.